README.md

```diff
@@ -23,6 +23,30 @@ module "ecs_cluster" {
 }
 ```
 
+## Repository
+Creates an Elastic Container Registry with associated pull & push policies.
+The created pull & push policies can be used as policy attachments on AWS roles.
+
+### Available variables:
+  * [`repository_name`]: String(required): The name of the ECR repository
+  * [`repository_description`]: String(required): The human readable description of the ECR repository
+  * [`expireAfter`]: Integer(optional): The amount of days after which untagged images expire (default: 30)
+
+### Output
+  * [`repository-url`]: The url to the ECR repository
```

Comment:
Can we use `_` on the output names as well? It'll be more consistent with the variable names